.gitignore

```diff
@@ -1,3 +1,4 @@
-.DS_Store
```

Comment:
This should be ignored at the [global `.gitignore`](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)